packages/vector_graphics_compiler/lib/src/svg/parser.dart

```diff
@@ -1645,6 +1640,33 @@ class SvgParser {
 
     throw StateError('Could not parse "$colorString" as a color.');
   }
+
+  @override
+  void addOpacityLayer(double opacity) {
+    _commands.add(DrawCommand(opacity, -1, DrawCommandType.opacity, null));
```

Comment:
This doesn't seem right to me.

The receiver end of the saveLayer will eventually want to know more than opacity, e.g. what blend mode to use or if there are other properties on the paint.

This should be adding a draw command with its own paint, even if that paint is solely composed of a Uint32 to express opacity.  The Object can be -1 I guess?

Reply:
https://github.com/dnfield/vector_graphics/pull/10#discussion_r823105486

Reply:
Done